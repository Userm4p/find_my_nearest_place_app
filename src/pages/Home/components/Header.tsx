import { useContext } from "react";
import { H1 } from "../../../components/H1/styled";
import { Input } from "../../../components/Input/styled";
import { CitiesContext } from "../context/CitiesContext";

export const Header = () => {
  const { handleChange, values } = useContext(CitiesContext);

  return (
    <>
      <H1>Find My Nearest Place App 🌍</H1>
      <Input
        placeholder="Type a city"
        onChange={handleChange}
        value={values.city}
        name="city"
      />
    </>
  );
};
