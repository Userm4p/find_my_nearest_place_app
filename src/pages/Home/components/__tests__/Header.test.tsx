import { render } from '@testing-library/react';
import { Header } from '../Header';

describe('Tests in Header', () => {
    test('should match snapshot',() => {
            const { container } = render(<Header />);
            expect(container).toMatchSnapshot();
    });
});