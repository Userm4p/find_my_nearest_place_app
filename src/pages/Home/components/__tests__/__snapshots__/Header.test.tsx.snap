// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in Header should match snapshot 1`] = `
<div>
  <h1
    class="sc-beySPh loJgJR"
  >
    Find My Nearest Place App 🌍
  </h1>
  <input
    class="sc-guDLey dTFHBz"
    name="city"
    placeholder="Type a city"
  />
</div>
`;
